applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -22,44 +23,59 @@ ZBUS_CHAN_DEFINE(sdu_ref_chan, struct sdu_ref_msg, NULL, NULL, ZBUS_OBSERVERS_EM
 
 #define HANDLE_INVALID 0xFFFF
 
-#define HCI_ISO_BUF_ALLOC_PER_CHAN 2
+#define HCI_ISO_BUF_PER_CHAN 2
+
+#if defined(CONFIG_BT_ISO_MAX_CIG) && defined(CONFIG_BT_ISO_MAX_BIG)
+#define GROUP_MAX (CONFIG_BT_ISO_MAX_BIG + CONFIG_BT_ISO_MAX_CIG)
+#elif defined(CONFIG_BT_ISO_MAX_CIG)
+#define GROUP_MAX CONFIG_BT_ISO_MAX_CIG
+#elif defined(CONFIG_BT_ISO_MAX_BIG)
+#define GROUP_MAX CONFIG_BT_ISO_MAX_BIG
+#else
+#error Neither CIG or BIG defined
+#endif
+
+#if defined(CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT)
+#define SUBGROUP_MAX (1 + CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT)
```

Comment:
Why +1 ?

Reply:
Because of unicast. I didn't want to do:

- only broadcast
- only unicast
- both

The only downside is that we are allocating a bit more memory, but I'm willing to discuss

Reply:
Write it more explicitly, checking all variables